### PR TITLE
Create path control

### PR DESCRIPTION
**Pull request by dtrucs**

### CODE REVIEW

File: src/stories/modules/Map/MapControls.stories.js [hunk 1]
```diff
@@ -62,10 +64,12 @@ const onChange = event => {
   action(title)(event.type);
 };
 
+const accessToken = 'pk.eyJ1IjoibWFraW5hY29ycHVzIiwiYSI6ImNqY3E4ZTNwcTFta3ozMm80d2xzY29wM2MifQ.Nwl_FHrWAIQ46s_lY0KNiQ';
```
Comment (jrmi) [inline src/stories/modules/Map/MapControls.stories.js:67]: PLS ;-)

File: src/modules/Map/Map/components/PathControl/PathControl.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    if (onPathUpdate) {
+      map.on('MapboxPathControl.update', onPathUpdate);
+    }
+
+    // eslint-disable-next-line no-param-reassign
+    map.pathControl = this.PathControl;
+
+    const prevOnRemove = this.PathControl.onRemove.bind(this.PathControl);
+
+    // this.PathControl.onRemove = function () {
```
Comment (jrmi) [inline src/modules/Map/Map/components/PathControl/PathControl.js:22]: Want to keep this ?

File: src/modules/Map/Map/components/PathControl/PathControl.test.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+  it('Should remove one listener', () => {
+    const controlInstance = new PathControl({ map, onPathCreate });
+
+    controlInstance.onRemove();
+    expect(map.off).toHaveBeenCalledTimes(1);
+    expect(map.off).toHaveBeenCalledWith('MapboxPathControl.create', onPathCreate);
+  });
+
+  it('Should remove 8 others listeners', () => {
```
Comment (jrmi) [inline src/modules/Map/Map/components/PathControl/PathControl.test.js:68]: 8 or 3 ?

**Review by jrmi — COMMENTED** [review]
Some minor things to review

File: src/modules/Map/Map/withMap.js [hunk 4]
```diff
@@ -19,7 +19,7 @@ export const DEFAULT_LAYER_TYPES_WEIGHT = {
   'fill-extrusion': 900,
 };
 
-const isDrawLayer = layer => layer.id.startsWith('gl-draw');
+const isDrawLayer = layer => layer.id.startsWith('gl-draw') || layer.id.startsWith('gl-pathControl');
```
Comment (jrmi) [inline src/modules/Map/Map/withMap.js:22]: Can you make it configurable by a props, default to the current list of to values ?

**Review by jrmi — APPROVED** [review]

**Review by jrmi — CHANGES_REQUESTED** [review]
Minor change if possible

File: src/modules/Map/Map/withMap.js (hunk 4 above)
Comment (dtrucs) [reply src/modules/Map/Map/withMap.js]: Workaround found by using `map.weightAttributionHooks`

**Review by jrmi — APPROVED** [review]

**Pull request merged**